Tests/Functional/Bundle/TestBundle/Resources/config/routing.yml

```diff
@@ -1,11 +1,19 @@
 request_body_param_converter:
     path:     /body-converter
-    defaults: { _controller: TestBundle:RequestBodyParamConverter:putPost }
+    defaults: { _controller: "TestBundle:RequestBodyParamConverter:putPost" }
```

Comment:
I got some errors .. but I think in the end it was probably something else .. so I guess this could be reverted